### PR TITLE
fix: remote with parent folders

### PR DESCRIPTION
**Pull request by jLopezbarb**
# Proposed changes

Fixes LAKE-191

We were using the path that the user was introducing in the --file. We should be using ManifestPath relative to the root project instead. This PR fixes that

## How to validate

1. Run cd api
1. Run okteto deploy -f ../okteto.yml  --remote

## CLI Quality Reminders 🔧

For both authors and reviewers:

- Scrutinize for potential regressions
- Ensure key automated tests are in place
- Build the CLI and test using the validation steps
- Assess Developer Experience impact (log messages, performances, etc)
- If too broad, consider breaking into smaller PRs
- Adhere to our [code style](https://github.com/okteto/okteto/blob/master/docs/code-style.md) and [code review](https://github.com/okteto/okteto/blob/master/docs/code-review.md) guidelines

### CODE REVIEW

**Review by ifbyol — COMMENTED** [review]
Should we add a test case that covers the scenario that was failing?

File: integration/deploy/manifest_test.go [hunk 1]
```diff
@@ -392,6 +392,55 @@ func TestDeployRemoteOktetoManifest(t *testing.T) {
...
+	}
+	require.NoError(t, commands.RunOktetoCreateNamespace(oktetoPath, namespaceOpts))
+	defer commands.RunOktetoDeleteNamespace(oktetoPath, namespaceOpts)
+	require.NoError(t, commands.RunOktetoKubeconfig(oktetoPath, dir))
+	c, _, err := okteto.NewK8sClientProvider().Provide(kubeconfig.Get([]string{filepath.Join(dir, ".kube", "config")}))
+	require.NoError(t, err)
+
+	require.NoError(t, createOktetoManifestWithDeployRemote(dir))
+	require.NoError(t, createAppDockerfileWithCache(dir))
+	require.NoError(t, os.Mkdir(parentFolder, 0700))
```
Comment (andreafalzetti) [inline integration/deploy/manifest_test.go:419]: I think integration tests are failing here

```
kubeconfig output:  ✓  Updated kubernetes context 'staging_okteto_dev/testdeployremoteparent-linux-1705691018-cindylopez' in '[/tmp/TestDeployRemoteOktetoManifestFromParentFolder4263263043/001/.kube/config]'
=== NAME  TestDeployRemoteOktetoManifestFromParentFolder
    manifest_test.go:419: 
        	Error Trace:	/home/circleci/project/integration/deploy/manifest_test.go:419
        	Error:      	Received unexpected error:
        	            	mkdir /tmp/TestDeployRemoteOktetoManifestFromParentFolder4263263043/001/app: file exists
        	Test:       	TestDeployRemoteOktetoManifestFromParentFolder
```

**Review by andreafalzetti — APPROVED** [review]
Tested and it works as expected, good work @jLopezbarb!

The integration tests are failing, before merging please have a look 🙏

**Comment by andreafalzetti** [comment]
After discussing on Slack, I am sharing here steps to reproduce another issue related to this. If you do:

1. Have a folder with several files (ie. I have a folder `dev` with all Okteto repos next to each other, including `movies-frontend`)
1. Go to that folder and run: `okteto deploy --remote -f movies-frontend/okteto.yml`
1. Notice how the CLI gets stuck in loading a huge context (for me it's GB).

**Pull request merged**